Source/SG/SG.cs

```diff
@@ -68,31 +73,27 @@ public struct PAENConfiguration
             public double CommandEnableTime_s;
             public double CommandDisableTime_s;
             public string PAEnableTriggerExportTerminal;
-            
-            public void SetDefaults()
+        }
+        public static PAENConfiguration GetDefaultPAENConfiguration()
+        {
+            return new PAENConfiguration
             {
                 //Default configuration is set for a DUT with a simple digital toggle high/low
                 //for PA Enable
-                PAEnableMode = PAENMode.Dynamic;
-                PAEnableTriggerExportTerminal = RfsgMarkerEventExportedOutputTerminal.Pfi0.ToString();
-                PAEnableTriggerMode = RfsgMarkerEventOutputBehaviour.Toggle;
-                CommandEnableTime_s = 0;
-                CommandDisableTime_s = 0;
-            }
+                PAEnableMode = PAENMode.Dynamic,
+                PAEnableTriggerExportTerminal = RfsgMarkerEventExportedOutputTerminal.Pfi0.ToString(),
+                PAEnableTriggerMode = RfsgMarkerEventOutputBehaviour.Toggle,
+                CommandEnableTime_s = 0,
+                CommandDisableTime_s = 0
+            };
         }
         #endregion
         public static void ConfigureInstrument(NIRfsg rfsgHandle, InstrumentConfiguration instrConfig)
         {
-
-            rfsgHandle.Arb.GenerationMode = RfsgWaveformGenerationMode.Script;
-            rfsgHandle.RF.PowerLevelType = RfsgRFPowerLevelType.PeakPower;
-
             rfsgHandle.RF.ExternalGain = -instrConfig.ExternalAttenuation_dBm;
             rfsgHandle.RF.Configure(instrConfig.CarrierFrequency_Hz, instrConfig.AverageInputPower_dBm);
 
             rfsgHandle.FrequencyReference.Source = RfsgFrequencyReferenceSource.FromString(instrConfig.ReferenceClockSource);
-            rfsgHandle.DeviceEvents.MarkerEvents[0].ExportedOutputTerminal =
-                RfsgMarkerEventExportedOutputTerminal.FromString(instrConfig.BurstStartTriggerExportTerminal);
 
             if (instrConfig.ShareLOSGToSA)
```

Comment:
where is the else case?! what happens if i want to disable LO sharing after I configured it to share the LOs once before

Reply:
Yikes, good point

Reply:
Fixed in ad04c39ac88e293b6dad2cf48805165c53808402